share/static/html/page.html

```diff
@@ -107,7 +107,7 @@ <h1 class="mdl-typography--display-2">
   {% block footer %}
   <footer class="mdl-mini-footer everware-footer">
       <div class="mdl-mini-footer__left-section">
-        Powered by <a href="https://jupyter.org/">Jupyter</a> and <a href="https://www.docker.com/">Docker</a>
+        v0.5.0 Powered by <a href="https://jupyter.org/">Jupyter</a> and <a href="https://www.docker.com/">Docker</a>
```

Comment:
idea is that this version is get defined within python package, so it should be possible to check it in a way like `print(everware.__version__)`. also it should be easy to increment. 
otherwise storing a version as a constant string deep down html would be a) hard to change b) impossible to query in other words this place should substitute template with variable value.